session.go

```diff
@@ -730,6 +760,21 @@ func (s *Session) muxFrameToLink(l *link, fr frames.FrameBody) {
 	debug.Log(2, "RX (Session %p): mux frame to link (%p): %s, %s", s, l, l.key.name, fr)
 }
 
+// transferEnvelope is used by senders to send transfer frames
+type transferEnvelope struct {
+	Ctx   context.Context
+	Frame frames.PerformTransfer
+	Sent  chan error
+}
+
+// frameBodyEnvelope is used by senders and receivers to send frames.
+// these frames come from the mux which is why there's no Sent channel.
```

Comment:
There is a `Sent` channel in here though. Old comment?

Reply:
Oops yes, from an earlier iteration (will remove).